Demo/MockVenmo/AppSwitcher.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+class AppSwitcher {
+
+    static var openVenmoURL: URL?
+
+    static var successURL: URL? {
+        var successComponents = openVenmoURL
+            .flatMap({ URLComponents(url: $0, resolvingAgainstBaseURL: false)})?
+            .queryItems?
+            .first(where: { $0.name == "x-success" })?
+            .value
+            .flatMap({ URLComponents(string: $0) })
```

Comment:
These are a bit confusing to me. Can you clarify what they're doing / needed for?